discussions/api/v1/serializers.py

```diff
@@ -7,8 +7,33 @@ class Meta:
         model = User.ReadState
 
 
-class UserSerializer(EmbeddedDocumentSerializer):
-    read_states = ReadStatesSerializer(many=True)
+class UserSerializer(DocumentSerializer):
 
     class Meta:
         model = User
+        fields = ('id', 'username', 'external_id', 'default_sort_key')
```

Comment:
actually it breaks if i do that.   i believe mongoengine maps "id" to "_id" when needed, or maybe i'm just getting lucky?